src/test/java/core/basesyntax/service/ReaderImplTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+import org.junit.Test;
+
+public class ReaderImplTest {
+    private final Reader reader = new ReaderImpl();
```

Comment:
initialize this service in a method annotated with Before or BeforeClass (depends on the version of junit)